src/test/java/uk/gov/hmcts/sptribs/judicialrefdata/JudicialServiceTest.java

```diff
@@ -65,7 +64,6 @@ void shouldPopulateUserDynamicList() {
         List<UserProfileRefreshResponse> responseEntity = List.of(userResponse1, userResponse2);
         var caseData = CaseData.builder().build();
 
-        //When
```

Comment:
Also if there is a chance to improve the respective code file, do not hesitate to fix it. (For example, var data type can be replaced by the respective data types in JudicialService etc etc)